tests/acceptance/onboarding-test.js

```diff
@@ -38,99 +40,99 @@ test('A user can onboard as expected', (assert) => {
 
   authenticateSession(application, { user_id: user.id });
 
-  visit('/');
+  indexPage.visit();
 
   andThen(() => {
     assert.equal(currentURL(), '/start/hello');
-    assert.equal(find('.start-actions button').is(':disabled'), true);
-    fillIn('input[name="firstName"]', 'Josh');
-    fillIn('input[name="lastName"]', 'Smith');
+    assert.ok(onboardingPage.startButton.isDisabled, 'start button disabled by default');
+    onboardingPage.firstName('Josh');
+    onboardingPage.lastName('Smith');
   });
 
   andThen(() => {
-    assert.equal(find('.start-actions button').is(':disabled'), false);
-    click('.start-actions button');
+    assert.notOk(onboardingPage.startButton.isDisabled, 'start button is enabled');
```

Comment:
Could do, but I'm not exactly sure on the syntax to bundle this up into a page object prop. the `attribute` selector doesn't seem to select a value, so I couldn't do `isEnabled: attribute('disabled', false)`, unless I can just make it CP?